index.php

```diff
@@ -20,13 +43,16 @@
                     <a href="#" class="list-item-link">Marketing</a>
                 </li>
             </ul>
+
+
             <div class="flex-container centered-vertically">
                 <div class="search-form-wrapper">
-                    <div class="search-form-field">
-                        <label>
-                            <input class="search-form-input" type="text" value="" placeholder="Search…" name="search">
-                        </label>
-                    </div>
+                    <form class="example" method="GET">
+                        <div class="search-form-field">
+                            <input class="search-form-input" type="text"
+                                   value="<?php if (isset($_GET['search'])) echo $_GET['search']; ?>"
```

Comment:
we are not escaping the `$_GET` variable, this might be dangeous

Reply:
fixed